[2025-04-18T00:28:51.525+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T00:28:51.531+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: triphobo_attractions_pipeline.load_geocoded_attractions_snowflake manual__2025-04-18T00:28:50.730363+00:00 [queued]>
[2025-04-18T00:28:51.533+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: triphobo_attractions_pipeline.load_geocoded_attractions_snowflake manual__2025-04-18T00:28:50.730363+00:00 [queued]>
[2025-04-18T00:28:51.533+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-04-18T00:28:51.538+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_geocoded_attractions_snowflake> on 2025-04-18 00:28:50.730363+00:00
[2025-04-18T00:28:51.540+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=279) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-18T00:28:51.541+0000] {standard_task_runner.py:72} INFO - Started process 287 to run task
[2025-04-18T00:28:51.542+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'triphobo_attractions_pipeline', 'load_geocoded_attractions_snowflake', 'manual__2025-04-18T00:28:50.730363+00:00', '--job-id', '439', '--raw', '--subdir', 'DAGS_FOLDER/attractions_dag.py', '--cfg-path', '/tmp/tmprbaoycm9']
[2025-04-18T00:28:51.543+0000] {standard_task_runner.py:105} INFO - Job 439: Subtask load_geocoded_attractions_snowflake
[2025-04-18T00:28:51.558+0000] {task_command.py:467} INFO - Running <TaskInstance: triphobo_attractions_pipeline.load_geocoded_attractions_snowflake manual__2025-04-18T00:28:50.730363+00:00 [running]> on host 56616e1d0c73
[2025-04-18T00:28:51.587+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='triphobo_attractions_pipeline' AIRFLOW_CTX_TASK_ID='load_geocoded_attractions_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T00:28:50.730363+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T00:28:50.730363+00:00'
[2025-04-18T00:28:51.588+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-18T00:28:51.649+0000] {attractions_dag.py:67} INFO - Downloading file from s3://bigdatafinal2025/clean/attractions_with_coords.csv
[2025-04-18T00:28:52.257+0000] {attractions_dag.py:71} INFO - CSV contains 855 rows and 14 columns
[2025-04-18T00:28:52.286+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.4, Python Version: 3.12.8, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-04-18T00:28:52.287+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-18T00:28:52.287+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-18T00:28:52.827+0000] {attractions_dag.py:82} INFO - Creating table TRIPHOBO_ATTRACTIONS with predefined schema...
[2025-04-18T00:28:53.034+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-04-18T00:28:53.035+0000] {attractions_dag.py:104} INFO - Uploading file to Snowflake stage...
[2025-04-18T00:28:54.071+0000] {attractions_dag.py:144} INFO - Copying data into Snowflake table...
[2025-04-18T00:28:55.069+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-04-18T00:28:55.234+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-04-18T00:28:55.236+0000] {attractions_dag.py:149} INFO - Snowflake table row count: 0
[2025-04-18T00:28:55.237+0000] {attractions_dag.py:154} WARNING - Row count mismatch! Expected 855, got 0
[2025-04-18T00:28:55.302+0000] {cursor.py:1166} INFO - Number of results in first chunk: 14
[2025-04-18T00:28:55.303+0000] {attractions_dag.py:159} INFO - Table columns: ['URL', 'DESCRIPTION', 'Travel Tips', 'Ticket Details', 'HOURS', 'How to Reach', 'Restaurants Nearby', 'IMAGE', 'CITY', 'Short Description', 'LATITUDE', 'LONGITUDE', 'PLACENAME', 'FORMATTEDADDRESS']
[2025-04-18T00:28:55.304+0000] {connection.py:789} INFO - closed
[2025-04-18T00:28:55.343+0000] {connection.py:795} INFO - No async queries seem to be running, deleting session
[2025-04-18T00:28:55.407+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-18T00:28:55.418+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-18T00:28:55.419+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=triphobo_attractions_pipeline, task_id=load_geocoded_attractions_snowflake, run_id=manual__2025-04-18T00:28:50.730363+00:00, execution_date=20250418T002850, start_date=20250418T002851, end_date=20250418T002855
[2025-04-18T00:28:55.438+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T00:28:55.449+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-18T00:28:55.449+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
